pkgs/tools/security/jadx/default.nix

```diff
@@ -16,97 +15,30 @@ let
   pname = "jadx";
   version = "1.4.7";
 
+in stdenv.mkDerivation (finalAttrs: {
+  inherit pname version;
```

Comment:
```suggestion
in stdenv.mkDerivation (finalAttrs: {
  pname = "jadx";
  version = "1.4.7";
```

nit

Reply:
can't be done as pname and version are used later